apps/storybook/src/overlays/Dialog.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogTitle,
+  Flex,
+} from "@optiaxiom/react";
+import { useState } from "react";
+
+const meta: Meta<typeof Dialog> = {
+  component: Dialog,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Dialog>;
+
+const DialogTemplate = ({
+  content,
+  size,
+}: {
+  content: string;
+  size: "lg" | "md" | "sm";
+}) => {
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <Flex>
+      <Button onClick={handleOpen}>Open Dialog</Button>
+      <Dialog onOpenChange={handleClose} open={open} size={size}>
```

Comment:
let's rename `onOpenChange` to `onClose`

Reply:
`onOpenChange` is Radix's own exposed function vai.

Reply:
i know - but since we're using this dialog in a fully controlled manner and we're not using the dialog trigger from radix - there is never a scenario where this function will be called with `true`. hence the naming `onClose` is much closer to its functionality now.